roles/etcd/meta/main.yml

```diff
@@ -18,3 +18,4 @@ galaxy_info:
 dependencies:
 - { role: os_firewall }
 - { role: etcd_common }
+- { role: etcd_server_certificates }
```

Comment:
btw, I might need you to submit a PR to implement the certificate work for this pr: https://github.com/openshift/openshift-ansible/pull/1503